src/resolve-dependency.ts

```diff
@@ -259,8 +259,21 @@ async function resolveRemappings(
 ): Promise<void> {
   if (job.conditions?.includes('browser')) {
     const { browser: pkgBrowser } = pkgCfg;
+    if (!pkgBrowser) {
+      return;
+    }
+    if (typeof pkgBrowser === 'string') {
+      return;
+    }
```

Comment:
```suggestion
```

I think you can remove this check since `typeof pkgBrowser === 'object'` doesnt have an `else` and will therefore return in the case of string